.builder/actions/crt-ci-prep-xcodebuild.py

```diff
@@ -0,0 +1,9 @@
+import Builder
+
+class CrtCiPrepXCodebuild(Builder.Action):
```

Comment:
Can we merge this to Main instead of iot branch? Swift SDK might need this as well. (I haven't reviewed the PR)